cmd/build/generate.go

```diff
@@ -59,7 +59,7 @@ func (Generate) code() error {
 		return fmt.Errorf("generating deep copy methods: %w", err)
 	}
 
-	err = shr.Run("controller-gen", "object", "paths=./internal/...")
+	err = sh.New(sh.WithWorkDir("apis")).Run("controller-gen", "object", "paths=./...")
```

Comment:
This changes functionality, is that intended?

Reply:
Sorry my bad. I restored the old behavior now.